src/core/mod.rs

```diff
@@ -38,6 +42,12 @@ mod crypto;
 // Private purely for organizational reasons; exported publicly above.
 mod jwk;
 
+///
+/// OpenID Connect Core device authorization response.
```

Comment:
the OIDC spec doesn't deal with the device flow, so let's just copy the docs from oauth2:
> Standard implementation of DeviceAuthorizationResponse which throws away extra received response fields.